### PR TITLE
Add permission check for list-database resource

### PR DESCRIPTION
**Pull request by yusukefs**
## What?

データベース一覧のリソースに api-permission-manager を通じた閲覧可能データベースのフィルタリング機能追加

## Why?

権限により閲覧できるデータベースを制限したい

## See also [Optional]

- Issue: https://github.com/dataware-tools/dataware-tools/issues/13

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/api-meta-store/pull/43#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapi-meta-store).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/api-meta-store/blob/main/OWNERS)~~ [yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by d-hayashi** [comment]
Related https://github.com/dataware-tools/api-permission-manager/pull/36

File: api/databases.py [hunk 1]
```diff
@@ -216,22 +216,28 @@ def _list_databases(sort_key: str,
     pql = parse_search_keyword(search_keyword, search_key)
     order_by = [(sort_key, 1)]
 
-    # Read
+    # Read all data
     handler.read(pql=pql, limit=per_page, offset=begin, order_by=order_by)
```
Comment (d-hayashi) [inline api/databases.py:220]: ↓で全部のデータを読み込めます
`handler.read(pql=pql, limit=None, offset=0, order_by=order_by)`

**Review by d-hayashi — COMMENTED** [review]
コメントの部分以外はLGTMです！

File: api/databases.py (hunk 1 above)
Comment (yusukefs) [reply api/databases.py:220]: ありがとうございます！！

Comment (yusukefs) [reply api/databases.py]: ↑修正しました！

**Review by d-hayashi — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/api-meta-store/pull/43#pullrequestreview-706100452" title="Approved">d-hayashi</a>*, *<a href="https://github.com/dataware-tools/api-meta-store/pull/43#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapi-meta-store).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/api-meta-store/blob/main/OWNERS)~~ [d-hayashi,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**